resources/wikia/modules/breakpointsLayout.js

```diff
@@ -0,0 +1,62 @@
+define('wikia.breakpointsLayout',  function() {
+	'use strict';
+
+	var WIDTH_OUTSIDE_MIN = 768,
+		CONTENT_BORDER = 1,
+		BREAKPOINT_LARGE_CONTENT_WIDTH = 1238,
+		BREAKPOINT_LARGE_PADDING_WIDTH = 30,
+		BREAKPOINT_MEDIUM_CONTENT_WIDTH = 1024,
+		BREAKPOINT_MEDIUM_PADDING_WIDTH = 20,
+		BREAKPOINT_MIN_CONTENT_WIDTH = 768,
+		BREAKPOINT_MIN_PADDING_WIDTH = 24,
+		RIGHT_RAIL_WIDTH_PLUS_SPACING = 320;
+
+	/**
+	 * Public API method for getting the maximum width to which an article can be resized
+	 *
+	 * @return {number} Maximum article width in pixels
+	 */
+	function getMinArticleWidth() {
```

Comment:
How about sticking to the convention you've established and naming it `getArticleMinWidth`? Also I'd move it next to `getArticleWidth`